nextcord/interactions.py

```diff
@@ -729,6 +743,9 @@ async def send_message(
         if delete_after is not None:
             await self._parent.delete_original_message(delay=delete_after)
 
+        if wait:
+            return await self._parent.original_message()
```

Comment:
Isnt there a wait=True thingy in send webhooks or does that not work here? I dont think we should be making requests without the knowledge of the user.

Reply:
Unfortunately, it seems like it won't work here. I've tried it, but when sending an interaction response, Discord responds with no content (204 response) meaning that it seems the only way to get the data is with an extra request